utils/tweet_query.py

```diff
@@ -67,7 +66,7 @@ def pull(self, query:str, output_dir: str,
 		next_token = None
 		num_collected = 0
 		for batch in batches:
-
+			start_time_req = time.time()
 			# Get tweet data from twitter api
 			try:
 				response = self.query_tweets(query, start_time = start_time, end_time = end_time, max_results = batch, next_token=next_token)
```

Comment:
Twitter has the /recent and /all endpoints (https://developer.twitter.com/en/docs/twitter-api/tweets/search/api-reference). Following this through, I see that we are using the /all endpoint here. But what if the user does not have the academic account? I think I will merge as is but make a a new issue.